### PR TITLE
Make necessary pages 

### PR DESCRIPTION
**Pull request by yuriko627**
Add 5 pages to support main functionalities of this Plasma Wallet 
- home page 
- send page 
- receive request page 
- exchange request page
- exchange order book page

Below pages will be added later 
- NFT collection page 
- NFT trade page 
- Plasma Block Explorer

### CODE REVIEW

File: components/MainDisplay.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <div className="main-display">
+      <div className="pages">{props.children}</div>
+      <div className="block-explorer-display">
+        <BlockExploer />
+      </div>
+      <style jsx>{`
+        .main-display {
+          display: flex;
+          height: 100%;s
```
Comment (tkmct) [inline components/MainDisplay.js]: typo?

File: components/Tabs.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import Dropdown from './Dropdown'
+import classNames from "classnames"
```
Comment (tkmct) [inline components/Tabs.js]: good!

File: pages/home.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+          </div>
+        <div className="send-receive-buttons">
+          <div className="send-button"><Link href="./send"><a className="send">Send</a></Link></div>
+          <div className="receive-button"><Link href="./receive"><a className="receive">Receive</a></Link></div>
+        </div>
+      </div>
+      <div className="transaction-history">
+        <div className="transaction-hisotry-title">Transaction History</div>
+        <div className="transaction-date">Nov 13, 2019</div>
+        <hr></hr>
```
Comment (tkmct) [inline pages/home.js]: `<hr />` is a self-closing tag!

File: components/Dropdown.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import Link from 'next/link'
+
+const Dropdown = ({ buttonName, items }) => {
+    return (
```
Comment (syuhei176) [inline components/Dropdown.js]: I missed it. Indent isn't same. Let's add lint:fix in later PR!

**Pull request merged**